networkit/cpp/io/test/IOBenchmark.cpp

```diff
@@ -25,13 +25,17 @@ namespace NetworKit {
 
 class IOBenchmark: public testing::Test {
 public:
-    static void convertToHeatMap(std::vector<bool> &infected, std::vector<double> &xcoords, std::vector<double> &ycoords, std::string filename, double resolution=1);
+    static void convertToHeatMap(std::vector<bool> &infected, std::vector<double> &xcoords,
+                                 std::vector<double> &ycoords, const std::string &filename,
+                                 double resolution = 1);
 };
 
 /**
  * This should actually not be in the benchmark but somewhere else. Can't figure out where yet.
  */
-void IOBenchmark::convertToHeatMap(std::vector<bool> &infected, std::vector<double> &xcoords, std::vector<double> &ycoords, std::string filename, double resolution) {
+void IOBenchmark::convertToHeatMap(std::vector<bool> &infected, std::vector<double> &xcoords,
+                                   std::vector<double> &ycoords, const std::string &filename,
+                                   double resolution) {
```

Comment:
Why is the default value of resolution removed?

Reply:
I don't see that in the removed line.

Reply:
Strange ... maybe some local caching. It was there in my version of the code.